MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/plugins/TrafficPluginActivity.java

```diff
@@ -0,0 +1,99 @@
+package com.mapbox.mapboxandroiddemo.examples.plugins;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+import com.mapbox.mapboxandroiddemo.R;
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.plugins.traffic.TrafficPlugin;
+
+/**
+ * Toggle the Mapbox Traffic plugin to display real-time traffic data on top
+ * of your map (not all regions supported at the moment).
+ */
+public class TrafficPluginActivity extends AppCompatActivity {
+
+  private MapView mapView;
+  private MapboxMap map;
+  private TrafficPlugin trafficPlugin;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    // Mapbox access token is configured here. This needs to be called either in your application
+    // object or in the same activity which contains the mapview.
+    Mapbox.getInstance(this, getString(R.string.access_token));
+
+    // This contains the MapView in XML and needs to be called after the access token is configured.
+    setContentView(R.layout.activity_traffic_plugin);
+
+    mapView = (MapView) findViewById(R.id.mapView);
+    mapView.onCreate(savedInstanceState);
+    mapView.getMapAsync(new OnMapReadyCallback() {
+      @Override
+      public void onMapReady(MapboxMap mapboxMap) {
+        TrafficPluginActivity.this.map = mapboxMap;
```

Comment:
You shouldn't need to clarify the variable location since it's a different name then the local `mapboxMap`, `TrafficPluginActivity.this.map` should just be `map = mapboxMap;`